loading ERM fails when preparing for maxfiltering if ERM contains EEG

pipeline calls to `maxwell_filter_prepare_emptyroom` seem like they'll always fail when the ERM contains EEG channels, because the pipeline always removes everything except MEG channels from `raw_ref`:

https://github.com/mne-tools/mne-bids-pipeline/blob/9d2548b5b23726fd944bec7ac751f6d58f07f082/mne_bids_pipeline/_import_data.py#L469-L480

Result looks like:

```
E   ValueError: DigMontage is only a subset of info. There are 60 channel positions not present in the DigMontage. The channels missing from the montage are:
E   
E   ['EEG001', 'EEG002', 'EEG003', 'EEG004', 'EEG005', 'EEG006', 'EEG007', 'EEG008', 'EEG009', 'EEG010', 'EEG011', 'EEG012', 'EEG013', 'EEG014', 'EEG015', 'EEG016', 'EEG017', 'EEG018', 'EEG019', 'EEG020', 'EEG021', 'EEG022', 'EEG023', 'EEG024', 'EEG025', 'EEG026', 'EEG027', 'EEG028', 'EEG029', 'EEG030', 'EEG031', 'EEG032', 'EEG033', 'EEG034', 'EEG035', 'EEG036', 'EEG037', 'EEG038', 'EEG039', 'EEG040', 'EEG041', 'EEG042', 'EEG043', 'EEG044', 'EEG045', 'EEG046', 'EEG047', 'EEG048', 'EEG049', 'EEG050', 'EEG051', 'EEG052', 'EEG053', 'EEG054', 'EEG055', 'EEG056', 'EEG057', 'EEG058', 'EEG059', 'EEG060'].
E   
E   Consider using inst.rename_channels to match the montage nomenclature, or inst.set_channel_types if these are not EEG channels, or use the on_missing parameter if the channel positions are allowed to be unknown in your analyses.
```

I'm hoping / assuming that just removing EEG channels from ERM also is the right way to go here (?)  If so I can open a PR.

--- a/docs/source/dev.md.inc
+++ b/docs/source/dev.md.inc
@@ -24,6 +24,7 @@
     2. cHPI filtering is now performed before movement compensation.
 
 - Fix bug where the ``config.proc`` parameter was not used properly during forward model creation (#1014 by @larsoner)
+- Fix bug where emptyroom recordings containing EEG channels would crash the pipeline during maxwell filtering (#1040 by @drammock)
 
 ### :books: Documentation
 

--- a/mne_bids_pipeline/_import_data.py
+++ b/mne_bids_pipeline/_import_data.py
@@ -467,6 +467,7 @@ def import_er_data(
         raw_ref.info["bads"] = bads
         raw_ref.info._check_consistency()
     raw_ref.pick_types(meg=True, exclude=[])
+    raw_er.pick_types(meg=True, exclude=[])
 
     if prepare_maxwell_filter:
         # We need to include any automatically found bad channels, if relevant.

